sci-libs/opencascade/opencascade-7.7.0-r1.ebuild

```diff
@@ -132,24 +133,16 @@ src_configure() {
 	fi
 
 	if use vtk; then
+		mycmakeargs+=(
+			-D3RDPARTY_VTK_DIR="${ESYSROOT}"/usr
+			-D3RDPARTY_VTK_LIBRARY_DIR="${ESYSROOT}"/usr/$(get_libdir)
+		)
 		if has_version ">=sci-libs/vtk-9.2.0"; then
-			mycmakeargs+=(
-				-D3RDPARTY_VTK_DIR="${ESYSROOT}"/usr
-				-D3RDPARTY_VTK_INCLUDE_DIR="${ESYSROOT}"/usr/include/vtk-9.2
-				-D3RDPARTY_VTK_LIBRARY_DIR="${ESYSROOT}"/usr/$(get_libdir)
-			)
+			mycmakeargs+=( -D3RDPARTY_VTK_INCLUDE_DIR="${ESYSROOT}"/usr/include/vtk-9.2 )
 		elif has_version ">=sci-libs/vtk-9.1.0"; then
-			mycmakeargs+=(
-				-D3RDPARTY_VTK_DIR="${ESYSROOT}"/usr
-				-D3RDPARTY_VTK_INCLUDE_DIR="${ESYSROOT}"/usr/include/vtk-9.1
-				-D3RDPARTY_VTK_LIBRARY_DIR="${ESYSROOT}"/usr/$(get_libdir)
-			)
+			mycmakeargs+=( -D3RDPARTY_VTK_INCLUDE_DIR="${ESYSROOT}"/usr/include/vtk-9.1 )
```

Comment:
We should really just use $(ver_cut) here I think and then we can drop a lot of this logic?

Reply:
Good idea, I'm gonna try it.

Reply:
Thanks!